### PR TITLE
HathiTrust links changed to OCN template

### PR DESCRIPTION
**Pull request by cdmo**
Will replace the ht struct with a single valued access level field. Requires https://github.com/psu-libraries/psulib_traject/pull/265

### CODE REVIEW

File: spec/models/hathi_links_spec.rb [hunk 1]
```diff
@@ -3,101 +3,89 @@
...
-      it 'generates a url to the catalog record with the etas text' do
-        expect(hathi_links).to match({
-                                       text: I18n.t('blackcat.hathitrust.etas_text'),
-                                       url: 'https://catalog.hathitrust.org/Record/12345?urlappend=%3Bsignon=swle:urn:mace:incommon:psu.edu',
-                                       additional_text: I18n.t('blackcat.hathitrust.etas_additional_text'),
-                                       etas_item: true
-                                     })
-      end
+    it 'generates a url to the catalog record with the etas text' do
+      document = { 'oclc_number_ssim': ['12345'],
```
Comment (banukutlu) [inline spec/models/hathi_links_spec.rb:79]: do we need this spec?

File: spec/models/hathi_links_spec.rb [hunk 2]
```diff
@@ -3,101 +3,89 @@
...
-                                       additional_text: nil,
-                                       etas_item: false
-                                     })
+      it 'generates a url with the public domain text when the record access is "allow' do
+        document = { 'oclc_number_ssim': ['12345'],
+                     'ht_access_ss': 'allow' }
+        hathi_link = SolrDocument.new(document).hathi_links
+
+        expect(hathi_link).to match({
+                                      text: 'Access digital copy through HathiTrust',
```
Comment (banukutlu) [inline spec/models/hathi_links_spec.rb:25]:
```suggestion
                                      text: I18n.t('blackcat.hathitrust.public_domain_text'),
```

File: spec/models/hathi_links_spec.rb (hunk 1 above)
Comment (banukutlu) [reply spec/models/hathi_links_spec.rb:79]: seems like similar to this spec https://github.com/psu-libraries/psulib_blacklight/pull/679/files?file-filters%5B%5D=.erb&file-filters%5B%5D=.rb&file-filters%5B%5D=.xml&w=1#diff-2f43653230e5e7a28792e91d306131d398675e5ce5051ecefd53496a7d12b774R19

**Review by banukutlu — CHANGES_REQUESTED** [review]
one question about specs

File: spec/models/hathi_links_spec.rb (hunk 1 above)
Comment (cdmo) [reply spec/models/hathi_links_spec.rb]: 🤦

**Review by banukutlu — APPROVED** [review]
👌

**Pull request merged**